### PR TITLE
solution for Customer Address default billing address Attribute Not U…

### PR DESCRIPTION
**Pull request by ghost**
Solution for Customer Address "default billing address" Attribute Not Used in Checkout #8777

In this solution I have set default billing address in checkout payment step.
### Description

 I have set default billing address in checkout payment step.There are some cases, one of them is  if customer has only one address and it is for default billing and default shipping then it will be checked as a "My billing and shipping address are same". second case is if billing and shipping address are different then in checkout payment step, default selected address would be default Billiing address.
### Fixed Issues (if relevant)

1. https://github.com/magento/magento2/issues/8777: Customer Address "default billing address" Attribute Not Used in Checkout #8777

### Manual testing scenarios

1. Log in to the frontend as a customer of a store.
2. Specify that an address is your default billing address is separate than shipping address.
3. Add any product to your cart.
4. Go to checkout.
5. Here default shipping address is selected by default , you can change it, Select any shipping method, then go to next step.
6. Here default billing address is selected and checkbox is unchecked.

### Contribution checklist
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [ ] All new or changed code is covered with unit/integration tests (if applicable)
 - [ ] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-cicd2** [comment]
[CLA assistant check](https://cla.dev.magento.com/magento/magento2?pullRequest=14474) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dev.magento.com/magento/magento2?pullRequest=14474) before we can accept your contribution.<br/><hr/>**Tejas** seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please [add the email address used for this commit to your account](https://help.github.com/articles/why-are-my-commits-linked-to-the-wrong-user/#commits-are-not-linked-to-any-user).

**Comment by larsroettig** [comment]
Hi @emiprotech,
currently  the JavaSciprt Tests are red please fix this see:
https://travis-ci.org/magento/magento2/jobs/360548261

best regards,

Lars

**Comment by barryvdh** [comment]
Thank you, this solves an important issue for me. 

Not sure about the javascript errors in the tests, it seems fine in my local setup.

**Comment by larsroettig** [comment]
Hi @barryvdh,
we can not merge if the tests are failed. 
Maybee if you have time you can finish this PR to solve this core issue for any merchant. 

best regards,

Lars

File: app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js [hunk 1]
```diff
@@ -92,6 +92,7 @@ define([
             }
             checkoutDataResolver.resolveShippingAddress();
 
+
```
Comment (larsroettig) [inline app/code/Magento/Checkout/view/frontend/web/js/view/shipping.js:95]: remove this empty line

**Review by larsroettig — CHANGES_REQUESTED** [review]

**Comment by sidolov** [comment]
@emiprotech , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for collaboration!

**Comment by barryvdh** [comment]
FYI, I had to modify vendor/magento/module-checkout/view/frontend/web/js/model/shipping-save-processor/default.js to not set it in the shipping step, otherwise it was still overriden.

**Pull request closed without merging**